steely/plugins/beer.py

```diff
@@ -57,6 +57,13 @@ def format_output(response):
 
 
 def main(bot, author_id, message, thread_id, thread_type, **kwargs):
+    global API_KEY
+    if 'BREWERYDB_API_KEY' in CONFIG:
+        API_KEY = CONFIG.BREWERYDB_API_KEY
```

Comment:
I'm guessing you've run this but I'm surprised accessing CONFIG works for both `in` like a dictionary and `.` like a module

Reply:
```
>>> import config
>>> config.TEST
'test'
>>> "TEST" in config
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: argument of type 'module' is not iterable
```
Tried it myself, I am confusion. But if you say it works then yolo

Reply:
It seemed to work, but I tried it now and it didn't so I fixed it.